src/halodrops/api/plotting.py

```diff
@@ -0,0 +1,19 @@
+from halodrops.plotting import quicklooks as ql
+
+
+# Access satellite data
+satellite_image = ql.get_satellite_data(**kwargs)
```

Comment:
The `**kwargs` are not defined in the file. Also, they are not needed as their aim is to pass on the non-default arg values, which we will have the main function do. So, for this function call and all others in this script, it should work to simply have empty parentheses.

Reply:
This also should solve the CI failing.

Reply:
Thanks for the comments! I put it there as a place holder but realize how it would cause an error... I fixed this and also adjusted the default extents.